src/schema/types/activityLog.type.ts

```diff
@@ -41,6 +41,20 @@ export const ActivityLogType = new GraphQLObjectType({
         }
       },
     },
+    title: {
+      type: GraphQLString,
+      description:
+        'The title of the page associated with the activity, if applicable.',
+      resolve: ({ metadata }) => {
+        try {
+          const parsedMetadata =
```

Comment:
@unai-reliefapp 
is there any case where the metadata can be a string?

Reply:
I guess not, but this is kept like that since the first integration of the activity model(check the url property too), guess it's something from the beginning that is no more useful, @TaiKamilla would know more I assume
KR, Unai